diff --git a/build.sh b/build.sh
index f0d3fd9..8118480 100755
--- a/build.sh
+++ b/build.sh
@@ -185,5 +185,7 @@ fi
 # see https://github.com/thinkshout/ts_recipes/blob/master/environment_setup.sh
 cd $DESTINATION
 sed -i '' 's/# RewriteBase \/$/RewriteBase \//g' ./.htaccess
+# remove the build directory, as it is not actually a profile.
+rm -rf profiles/$PROJECT
 
 printf "\nBuild script complete.\n"
